x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/process/AnalyticsResultProcessor.java

```diff
@@ -273,4 +272,9 @@ private void setAndReportFailure(Exception e) {
         failure = "error processing results; " + e.getMessage();
         auditor.error(analytics.getId(), "Error processing results; " + e.getMessage());
     }
+
+    private void processMemoryUsage(MemoryUsage memoryUsage) {
```

Comment:
Could you make the code more consistent by either:
1. inlining this method again

or

2. making other result types (e.g.: `OutlierDetectionStats`) also have similar private methods

?

Reply:
Right, I extracted this so that I also add the audit message but then left it for a follow up PR. I don't think it's worth extracting methods when it's a single liner but it makes sense to do so when there are more lines. Would it be fine to do this in the follow up PR I'll do anyway? Just so we don't lose CI cycles on this one.

Reply:
SGTM